pai-management/bootstrap/grafana/grafana-configuration/pai-clusterview-dashboard.json.template

```diff
@@ -388,7 +388,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "100 - avg (irate(node_cpu_seconds_total{mode=\"idle\"}[5m])) * 100",
+              "expr": "100 - avg (irate(node_cpu_seconds_total{mode=\"idle\"}[{{clusterinfo['prometheusinfo']['scrape_interval']}}0s])) * 100",
```

Comment:
Is there an extra '0'? -- {{clusterinfo['prometheusinfo']['scrape_interval']}}**0**s

Reply:
Yes, as we can't do arithmetic in this grafana metrics, so I did a hack that add a "0", let it times 10.